docker-storage-setup.sh

```diff
@@ -55,13 +55,13 @@ set -e
 # Note: lvm2 version should be same or higher than lvm2-2.02.112 for lvm
 #       thin pool functionality to work properly.
 POOL_LV_NAME="docker-pool"
-DOCKER_ROOT_LV_NAME="docker-root-lv"
-DOCKER_ROOT_DIR="/var/lib/docker"
+ROOT_LV_NAME="docker-root-lv"
+ROOT_DIR="/var/lib/docker"
 
-DOCKER_STORAGE="/etc/sysconfig/docker-storage"
+STORAGE="/etc/sysconfig/docker-storage"
 STORAGE_DRIVERS="devicemapper overlay overlay2"
 
-DOCKER_METADATA_DIR="/var/lib/docker"
+METADATA_DIR="/var/lib/docker"
```

Comment:
The whole purpose of this METADATA_DIR was to check if docker already has /var/lib/dokcer/devicemapper dir which tells it that docker has been using loop devices and it errors out in this case.

This is another case of docker-storage-setup probing metadata of caller which is completely wrong. We just did it for better user experience. So in long term I would like to get rid of this functionality and not expose this knob to OCID and other tools at all. Before they call docker-storage-setup, they need to make sure they are not using loop devices. Its none of the docker-storage-setup business.